### PR TITLE
tempest: Install monasca packages with tempest tests

### PR DESCRIPTION
**Pull request by nicolasbock**

### CODE REVIEW

File: chef/cookbooks/tempest/recipes/install.rb [hunk 1]
```diff
@@ -35,3 +35,7 @@
  "heat", "ceilometer", "sahara"].each do |component|
   package "python-#{component}client"
 end
+
+["api", "log-api"].each do |component|
+  package "python-monasca-#{component}"
```
Comment (dirkmueller) [inline chef/cookbooks/tempest/recipes/install.rb]: These packages only exist on x86 and should only be installed if Monsaca is being deployed

Comment (nicolasbock) [inline chef/cookbooks/tempest/recipes/install.rb]: Please have another look.

**Comment by toabctl** [comment]
commit msg should start with ```tempest: ...```

**Comment by nicolasbock** [comment]
@toabctl done

**Pull request merged**